trustwallet/wallet-core
# fixed the lack of installation of the rust library in cmake, on which…

… the main library depends

## Description

Before making changes, when calling the install,`make install` for example, the library that the project depended on was not installed. As a result of the changes, the installation was corrected, so the library built using Rust will also be installed without violating the integrity.

## How to test

After building the project, call
`make install`
and make sure that the rust(ibwallet_core_rs.a) library has also been installed.

## Types of changes

СMake changes. Installation step

## Checklist



- [] Create pull request as draft initially, unless its complete.

If you're adding a new blockchain
